client/verta/verta/_tracking/experimentruns.py

```diff
@@ -86,7 +94,7 @@ def as_dataframe(self):
 
             data.append(run_data)
 
-        return pd.DataFrame(data, columns=['id'] + sorted(list(columns)))
+        return pd.DataFrame(data, columns=['id'] + sorted(list(columns))).set_index('id')
```

Comment:
Can we do it at creation time to avoid re-index?

Reply:
Done.